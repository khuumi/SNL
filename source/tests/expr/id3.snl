asdf123_jkl14asdklf