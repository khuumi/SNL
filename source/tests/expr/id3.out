Id Global asdf123_jkl14asdklf